tests/conda/default-env/postBuild

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 # install pytest with conda in the default env (should be $KERNEL_PYTHON_PREFIX)
-conda install -yq pytest
+/tmp/bin/micromamba install -yq pytest -p $KERNEL_PYTHON_PREFIX
```

Comment:
Can this be left as-is? The test should still pass calling `conda` without `-p`, otherwise this is breaking things.

Reply:
Done